sdk/core/src/services/WalletStorageService.ts

```diff
@@ -371,15 +365,137 @@ export default class WalletStorageService {
     throw new SdkErrorFromCode('COR-23', { id })
   }
 
-  async deleteCredential(id?: string, credentialIndex?: number): Promise<void> {
-    if ((credentialIndex !== undefined && id) || (!id && credentialIndex === undefined)) {
-      throw new SdkErrorFromCode('COR-1', {
-        errors: [{ message: 'should pass either id or credentialIndex and not both at the same time' }],
+  public async saveCredentials(audienceDid: string, credentials: SignedCredential[], storageRegion?: string) {
```

Comment:
So `WalletStorageService` gets some of the auth info as constructor params (as two params, `keysService` and `didAuthService` with somewhat duplicated data, and some as method params, on every method call.

But `audienceDid` should not change between different calls to the same instance of `WalletStorageService`. So why not to pass it to the constructor as well, at least?

Reply:
I see, `did` created once during first get. Good point!